packages/server/src/channels/messenger/channel.ts

```diff
@@ -89,7 +89,9 @@ export class MessengerChannel extends Channel<MessengerConduit> {
     const [, hash] = signature.split('=')
     const expectedHash = crypto.createHmac('sha1', conduit.config.appSecret).update(buffer).digest('hex')
     if (hash !== expectedHash) {
-      this.logger.error("Couldn't validate the request signature. Make sure your appSecret is properly configured.")
+      // TODO: can we just throw the error with the message in it instead??
+      const e = new Error()
+      this.logger.error(e, "Couldn't validate the request signature. Make sure your appSecret is properly configured")
```

Comment:
Yeah, I think you should do something like this:

```suggestion
      const message = "Couldn't validate the request signature. Make sure your appSecret is properly configured"
      this.logger.error(new Error(message), message)
```

So at least you have the stack trace and the error message if needed!

Reply:
I'll just log the error, so we avoid having the same message twice